benchexec/tools/metaval.py

```diff
@@ -0,0 +1,85 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2019  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+import os
+
+
+class Tool(benchexec.tools.template.BaseTool):
+
+    REQUIRED_PATHS = ["CPAchecker", "esbmc"]
```

Comment:
```suggestion
    REQUIRED_PATHS = util.flatten(TOOL_TO_PATH_MAP.values())
```
Would avoid duplication of required paths (order of fields needs to be swapped, of course).

Or even simplify `TOOL_TO_PATH_MAP` by making its values single strings instead of lists and remove the `flatten`.